src/api_client/api_client.py

```diff
@@ -1,110 +1,76 @@
-                                          limit=1000,
-                                          startTime=start_time,
-                                          endTime=end_time)
-        return response
-
     def get_current_price(self, currency_to_buy, currency_to_sell):
         return self.client.get_symbol_ticker(symbol=f'{currency_to_buy}{currency_to_sell}')['price']
 
     def trade(self, currency_to_buy, currency_to_sell, action, quantity):
         response = self.client.create_order(symbol=f'{currency_to_buy}{currency_to_sell}',
-                                            quantity=super().round(quantity),
+                                            quantity=round(quantity),
                                             side=action,
                                             type='MARKET')
         return response['orderId']
 
     def view_order(self, order_id):
-        response = self.client.get_order(orderId=order_id)
-        return response
+        pass
 
     def view_account(self):
         pass
 
 
 
-class CoinbaseClient(APIClient):
-    def __init__(self, key, secret, passphrase, api_url, **kwargs):
-        super().__init__(**kwargs)
-        self.client = cbpro.AuthenticatedClient(key=key,
-                                                b64secret=secret,
-                                                passphrase=passphrase,
-                                                api_url=api_url)
-
-    def trade(self, action, limitPrice, quantity):
-        if action == 'buy':
-            response = self.client.buy(
-                price=limitPrice,
-                size=self.round(quantity),
-                order_type='limit',
-                product_id=f'{config_coinbase.CURRENCY_TO_BUY}-{config_coinbase.CURRENCY_TO_SELL}',
-                overdraft_enabled=False
-            )
-            return response['id']
-
-        elif action == 'sell':
-            response = self.client.sell(
-                price=limitPrice,
-                size=self.round(quantity),
-                order_type='limit',
-                product_id=f'{config_coinbase.CURRENCY_TO_BUY}-{config_coinbase.CURRENCY_TO_SELL}',
-                overdraft_enabled=False
-            )
-            return response['orderId']
-
-        def view_order(self, order_id):
-            response = self.client.get_order(orderId=order_id)
-            return response  # {'status': response['status'], 'fee' : response['fee'], 'done_reason' : response['done_reason']}
-
-        def viewAccounts(self, accountCurrency):
-            accounts = self.client.get_accounts()
-            account = list(filter(lambda x: x['currency'] == accountCurrency, accounts))[0]
-            return accounts
-
-        def getCurrentPriceOfCurrency(self):
-            tick = self.client.get_product_ticker(
-                product_id=f'{config_coinbase.CURRENCY_TO_BUY}-{config_coinbase.CURRENCY_TO_SELL}')
-            return tick['bid']
+# class CoinbaseClient:
```

Comment:
You can directly remove it, no need to comment it since we already have the code in the github story.